docs/_data/colors.yml

```diff
@@ -138,114 +144,130 @@ group:
   key: status
   colors:
   - colorname: 1
-    colorvalue: 107F3E
-    colorhsl: hsl(145, 78%, 28%)
+    colorvalue: 107E3E
+    colorhsl: hsl(147, 85.3%, 26.7%)
     textcolor: FFFFFF
     bordercolor: 107F3E
 
   - colorname: 2
     colorvalue: E9730C
-    colorhsl: hsl(28, 90%, 48%)
-    textcolor: FFFFFF
+    colorhsl: hsl(28, 90.2%, 48%)
+    textcolor: 000000
     bordercolor: E9730C
 
   - colorname: 3
     colorvalue: BB0000
-    colorhsl: hsl(0, 100%, 37%)
+    colorhsl: hsl(0, 100%, 36.7%)
     textcolor: FFFFFF
     bordercolor: EE0000
 
   - colorname: 4
-    colorvalue: 6D7678
-    colorhsl: hsl(191, 5%, 45%)
+    colorvalue: 6A6D70
+    colorhsl: hsl(210, 2.8%, 42.7%)
     textcolor: FFFFFF
     bordercolor: 6D7678
 
+  - colorname: 5
+    colorvalue: 0A6ED1
+    colorhsl: hsl(210, 90.9%, 42.9%)
+    textcolor: FFFFFF
+    bordercolor: 0A6ED1
+
 
 # Accent
 - name: Accent
   key: accent
-  description: These colors are optimized for two different uses. The first ten colors meet a contrast standard of 3:1 and are best used for differentiating between similar items, e.g., in a calendar, legend, or pills. The last six colors meet a contrast standard or 4.5:1 and best used when text is important, e.g., identifiers and avatars.
+  description: These colors are optimized for two different uses. The first ten colors meet a contrast standard of 3:1 and are best used for differentiating between similar items, e.g., in a calendar, legend, or pills. The last four colors meet a contrast standard or 4.6:1 and best used when text is important, e.g., identifiers and avatars.
```

Comment:
Not sure where this orignated from.

when it says they meet contrast standard of 3:1, what is the color it is being contrasted with? White? The base color it is an accent on? 3:1 make no sense out of context.

Same is true for 4.6:1

Additionally, So while it is admirable we are  4.6:1 on the last 4 values, A11y standards are shooting for 4.5:1 ratio or higher. If I am customer searching trough mountains of documentation, I might be tempted to search for text `4.5`, and the value `4.6` might get lost in the weeds. Not worth deviating here.

Reply:
<img width="303" alt="Screen Shot 2019-03-16 at 10 07 21 PM" src="https://user-images.githubusercontent.com/4380815/54484212-e726ed80-4837-11e9-9cb4-c2fdb2c0b068.png">

Reply:
@droshev : ah it's against `#FFFFFF`. Worth calling out. 

as far as 4.6 is concerned, i would only think this was chosen because there is disagreement if 4.5 itself is passing, or that it needs to be higher than 4.5. But if that were the case, they should have gone with 3.1:1 for the lighter variations. 

But nothing for you do about it here.